waspc/src/Wasp/AI/OpenAI/ChatGPT.hs

```diff
@@ -13,13 +13,19 @@ module Wasp.AI.OpenAI.ChatGPT
 where
 
 import Control.Arrow ()
+import Control.Concurrent (threadDelay)
+import Control.Monad.IO.Class (MonadIO (liftIO))
 import Data.Aeson ((.=))
 import qualified Data.Aeson as Aeson
 import Data.ByteString.UTF8 as BSU
 import Data.Text (Text)
 import GHC.Generics (Generic)
+import qualified Network.HTTP.Conduit as HTTP.C
 import qualified Network.HTTP.Simple as HTTP
+import UnliftIO (MonadUnliftIO)
+import UnliftIO.Exception (catch, throwIO)
```

Comment:
Is there a reason you use `catch` and `throwIO` from `unliftio` instead of `base`?

Reply:
I read in article by FPComplete that it is better to use the one from unliftio so I use that one through the codebase! Although I can't say I have a super strong understanding of the matter. What is your opinion on this?

Here is the article: https://www.fpcomplete.com/haskell/tutorial/exceptions/ .

Reply:
I gave that article a read over the weekend, lots of interesting stuff! Seems like there are several good reasons to use the ones from unliftio instead of from base, so lets stick with these ones.